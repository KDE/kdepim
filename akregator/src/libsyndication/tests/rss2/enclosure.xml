<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0" xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd">
    <channel>
        <item>
            <title>An item with enclosure</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
        </item>
        <item>
            <title>An item with enclosure and itunes duration (&quot;20:20&quot; == 1220 seconds)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>20:20</itunes:duration>
        </item>
        <item>
            <title>An item with enclosure and itunes duration (&quot;20:20:00&quot; == 73200 seconds)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>20:20:00</itunes:duration>
        </item>
        <item>
            <title>An item with enclosure and itunes duration (&quot;20:20:20&quot; == 73220 seconds)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>20:20:20</itunes:duration>
        </item>
        <item>
            <title>An item with enclosure and invalid itunes duration (&quot;-50&quot;)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>-50</itunes:duration>
        </item>
        <item>
            <title>An item with enclosure and invalid itunes duration (&quot;50 minutes 4 seconds&quot;)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>50 min 4 sec</itunes:duration>
        </item>
        <item>
            <title>An item with enclosure and itunes duration in seconds (&quot;1220&quot;)</title>
            <enclosure url='http://www.example.com/somemp3.mp3' length='12345678' type='audio/mpeg'/>
            <itunes:duration>1220</itunes:duration>
        </item>

        <item>
            <title>An item with two enclosures and itunes duration (1220).</title>
            <enclosure url='http://www.example.com/somefile.mp3' length='12345678' type='audio/mpeg'/>
            <enclosure url='http://www.example.com/somefile.ogg' length='10345678' type='audio/vorbis'/>
            <itunes:duration>1220</itunes:duration>
        </item>
    </channel>
</rss>
