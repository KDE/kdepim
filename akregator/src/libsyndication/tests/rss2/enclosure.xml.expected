# Feed begin ######################
# Item begin ######################
id: #hash:92d65022dd9d6efa730a571484082689#
title: #An item with enclosure#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:bad09b8bfa7e11acef2eda32c96f9fa4#
title: #An item with enclosure and itunes duration (&quot;20:20&quot; == 1220 seconds)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
duration: #1220 (0:20:20)#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:b7db210aa2da89d61c3ddbd7941a378c#
title: #An item with enclosure and itunes duration (&quot;20:20:00&quot; == 73200 seconds)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
duration: #73200 (20:20:0)#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:c6f405d2d831f64d71af4be588bd1354#
title: #An item with enclosure and itunes duration (&quot;20:20:20&quot; == 73220 seconds)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
duration: #73220 (20:20:20)#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:901ea52b518047436765073bc0b5b8ea#
title: #An item with enclosure and invalid itunes duration (&quot;-50&quot;)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:ce79e6f6d6df3185f1f7b7ce1d280a3d#
title: #An item with enclosure and invalid itunes duration (&quot;50 minutes 4 seconds&quot;)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:9d2b5ff09afe283a6ad4d24a40440da3#
title: #An item with enclosure and itunes duration in seconds (&quot;1220&quot;)#
# Enclosure begin #################
url: #http://www.example.com/somemp3.mp3#
type: #audio/mpeg#
length: #12345678#
duration: #1220 (0:20:20)#
# Enclosure end ###################
# Item end ########################
# Item begin ######################
id: #hash:516f779e8dcb035c4dba2d6713441b32#
title: #An item with two enclosures and itunes duration (1220).#
# Enclosure begin #################
url: #http://www.example.com/somefile.mp3#
type: #audio/mpeg#
length: #12345678#
duration: #1220 (0:20:20)#
# Enclosure end ###################
# Enclosure begin #################
url: #http://www.example.com/somefile.ogg#
type: #audio/vorbis#
length: #10345678#
duration: #1220 (0:20:20)#
# Enclosure end ###################
# Item end ########################
# Feed end ########################

